### PR TITLE
Prow migration for console

### LINKED ISSUE #353: Prow pipeline for kyma/console
- Migrate CI from Jenkins into Prow job, and make prow as master CI
- Check if we need to update documentation

### PR DESCRIPTION
**Pull request by y-kkamil**
**Description**

Added Makefile for prow ci.

**Related issue(s)**
Resolves #353

### CODE REVIEW

File: core/Makefile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+TAG = $(DOCKER_TAG)
+
+ci-pr: install-dependencies lint test build validate build-image push-image
+ci-master: install-dependencies lint test build validate build-image push-image
+ci-release: install-dependencies lint test build validate build-image push-image
+
+install-dependencies:
+	npm ci
+
+lint:
```
Comment (michal-hudy) [inline core/Makefile]: Jenkins expects that `validate` target is defined: https://github.com/kyma-project/console/blob/master/core/Jenkinsfile#L57
It is also expected by `ci*` targets, see lines 5,6,7 ;)


Change it here or in `jenkinsfile`

Comment (y-kkamil) [reply core/Makefile]: fixed

**Comment by michal-hudy** [comment]
@jesusreal Branch protection will be configured by Prow, when we will migrate. Right now Prow is not production ready, so it should not block PR. Also, as you can see it didn't send status to GH.

**Pull request merged**